### DISCUSSION TITLE
代码的复制功能在iOS/macOS上异常，在android正常

### DISCUSSION THREAD

**Discussion by lhxone** [liuyib/hexo-theme-stun#181] [Q&A（反馈问题、提问、报告 Bug 等）] [answered]
![image](https://user-images.githubusercontent.com/55795964/113713149-908bf500-9719-11eb-89ed-543d0280f88c.png)

**Comment by lhxone** [comment]
在macOS的Safari环境中异常，但是Chrome正常可用

**Comment by liuyib** [comment, marked as answer]
这是因为主题实现复制时，用的一些 API 兼容性不太好，以后这个问题会修复的，感谢反馈 ❤️
